docs/src/data/SDKInfo.ts

```diff
@@ -3,7 +3,7 @@
 export default {
   js: {
     name: "JS SDK",
-    version: "1.6.1",
+    version: "1.6.2",
```

Comment:
It looks like you ran the `yarn bump-sdk-version` script.  Instead, you should run `yarn generate-api-types`.  

The change you are making does not require any version bumps.  Can you revert all of those changes from this PR?

Reply:
Yes!